sources/api/netdog/src/cli/primary_interface.rs

```diff
@@ -0,0 +1,13 @@
+use super::{primary_interface_name, print_json, Result};
+use argh::FromArgs;
+
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "primary-interface")]
+/// Return the current IP address
+pub(crate) struct PrimaryInterfaceArgs {}
+
+/// Return the current IP address as JSON (intended for use as a settings generator)
```

Comment:
nit: this comment seems like a copy/paste artifact, since I don't really see how we could use a settings generator for primary interface.
```suggestion
```